controller/python/README.md

```diff
@@ -86,3 +86,37 @@ Here is the usage:
     SPACE:    Toggle logging
     ESC:      Quit
 ```
+
+# Python Controller with Remote RTP Video Stream and TCP/IP control
+This python program allows to control the robot from a [joystick controller](joystick_controller.py) (you can also make this work with the keyboard controller), and receive [video stream](camera.py) from the camera via both mobile network connection and wifi connection. This program also allows to toggle data logging and autopilot on the robot's phone remotely. The program can run on computers that are necessarily in the same network as the robot's phone. It was developed and tested on Aruidno Uno and a Linux computer.
+
+## Dependencies
+In addition to the dependencies above, you need to first build OpenCV with GStreamer, which will be used to decode RTP packets for video stream. And then pip install the wheel file generated by the opencv build process. 
+
+## Network Connection
+1. Go to `Settings` and select `RTP` as the `Stream Mode`. 
+2. Then type in your IP and port numbers for control and video stream. (Remote control uses TCP/IP, and video stream uses RTP)
+3. Restart the app to load the new settings
+
+**If your android devide uses mobile network, and connects to the PC via Internet, then you need to do port forwarding on your router for both ports of remote control and remote video stream**. **!!!This must be done before you connect the android devide with the PC!!!**
+
+## Using this remote controller
+First connect you joystick with the computer.
+Then, run the controller
+```
+python joystick_controller.py
+```
+
+Once robot app successfully connects to Python script, then run the video stream: 
+```
+python camera.py
+```
+
+Controller mapping
+```
+▲: autodrive
```

Comment:
On the BT joystick, autopilot is on R1, data logging on X. Let's be consistent. Triangle, square and circle are for the indicators.